File: week1/day5/algo-pad-array/js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, value=null) => {
    
    if (array.length >= minSize) {
        // console.log('working')
        return array
    } else {
        let add_length = minSize - array.length
        // console.log(add_length)
        
        for (let i = 0; i < add_length; i++) {
            array.push(value)
        }
    }
    // console.log(array)
    return array




exports.pad = pad;

File: week1/day5/algo-pad-array/python/pad_array.py
#REMEMBER TO PSEUDOCODE
from unittest import result


def pad(array, min_size, value = None):
    

    if len(array) >= min_size:
        return array[:min_size]

    num_to_add = min_size - len(array)

    for i in range(num_to_add):
        array.append(value)

    return array